2-mandatory/1-writers.js

```diff
@@ -61,6 +61,9 @@ Exercise 1:
 */
 function logAllWriters() {
   // write your code to log all writers here
+  for (let x of writers) {
+    console.log(`Hi, my name is ${x.firstName} ${x.lastName}. I am ${x.age} years old, and work as a ${x.occupation}.`);
```

Comment:
Hi Anosh, nice work. I spoke about this with Jihad. We can understand what x does in the function but what if you're trying to explain the code to a total beginner? I suggest using a naming that can be more understandable to people of all levels, to understand. In that way, no one gets confused trying to understand the code.